### PR TITLE
feat(dashboard,types): split damaged activity from received

### PR DESCRIPTION
**Pull request by riqwan**
what:

- split damaged activity from received
- adds a popover to view return details

Screenshot 2024-08-28 at 23 23 10

RESOLVES CC-363

### CODE REVIEW

**Review by carlos-r-l-rodrigues — APPROVED** [review]

File: packages/admin-next/dashboard/src/routes/orders/order-detail/components/order-summary-section/return-info-popover.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      </Popover.Trigger>
+
+      <Popover.Content
+        align="center"
+        side="top"
+        className="bg-ui-bg-component focus-visible:outline-none p-2"
+      >
+        <div className="flex flex-col">
+          <Badge size="2xsmall" className="mb-2" rounded="full">
+            #{orderReturn.id.replace("return_", "")}
```
Comment (fPolic) [inline packages/admin-next/dashboard/src/routes/orders/order-detail/components/order-summary-section/return-info-popover.tsx]: suggestion: can we keep the same number of characters consistent across the page so users can easily identifies the same return

File: packages/admin-next/dashboard/src/routes/orders/order-detail/components/order-summary-section/return-info-popover.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+            {" · "}
+            {formatDate(orderReturn.requested_at)}
+          </Text>
+
+          <Text size="xsmall">
+            <span className="text-ui-fg-subtle">
+              {t(`orders.returns.itemReceived`)}
+            </span>
+            {" · "}
+            {formatDate(orderReturn.received_at)}
```
Comment (fPolic) [inline packages/admin-next/dashboard/src/routes/orders/order-detail/components/order-summary-section/return-info-popover.tsx]: suggestion: add fallback to "-" if for some reason this is `null`

File: packages/admin-next/dashboard/src/i18n/translations/en.json [hunk 3]
```diff
@@ -924,8 +924,12 @@
...
       "returnableQuantityLabel": "Returnable quantity",
       "refundableAmountLabel": "Refundable amount",
-      "returnRequestedInfo": "{{requestedItemsCount}}x item return requested",
-      "returnReceivedInfo": "{{requestedItemsCount}}x item return received",
+      "returnRequestedInfo": "{{requestedItemsCount}}x items return requested",
+      "returnReceivedInfo": "{{requestedItemsCount}}x items return received",
+      "itemReceived": "Items received",
+      "returnRequested": "Return requested",
+      "damagedItemReceived": "Damaged items received",
+      "damagedItemsReturned": "{{quantity}}x Damaged items returned",
```
Comment (fPolic) [inline packages/admin-next/dashboard/src/i18n/translations/en.json]: suggestion/nit: can "Damaged" be lower first letter on L:932, it looks a bit off next to the return received line

**Review by fPolic — APPROVED** [review]
LGTM - nice!

**Pull request merged**